tests/subsys/mgmt/mcumgr/mcumgr_client/src/img_gr_stub.c

```diff
@@ -242,7 +242,7 @@ void img_upload_init_verify(struct net_buf *nb)
 	image = UINT32_MAX;
 
 	rc = zcbor_map_decode_bulk(zsd, list_res_decode, ARRAY_SIZE(list_res_decode), &decoded);
-	if (rc || data.len == 0 || offset == SIZE_MAX || image != TEST_IMAGE_NUM) {
+	if (rc) {
```

Comment:
Why? The `zcbor_map_decode_bulk` will return ok even if the decoded fields are not found.
The check here tests if frame, that has been correctly decoded, contains all needed data.